build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
+  "master": {
+    "nugetPush": [
+        {
+            "kind": "PerBuildPreRelease", 
```

Comment:
Is that a process we'd be doing right now though? We've published one RTM package for 2.3 line, but would we be publishing RTM for each build we make since then?

(To be clear: question here is not thinking that's a _bad_ thing. Just want 100% clarity so I can tell other people what to do when they're editing this.)